scripts/repl.js

```diff
@@ -31,8 +31,10 @@
   var getQuery = function() {
     var query = window.location.hash.replace(/^\#\?/, '');
     return _.transform(query ? query.split('&') : null, function(result, val) {
-      val = val.split('=');
-      result[val[0]] = decodeURIComponent('' + val[1]);
+      var equalsIndex = val.indexOf('=');
+      var name = val.substring(0, equalsIndex);
+      var value = val.substring(equalsIndex + 2);
```

Comment:
:/ thx for fixing it!

On Tuesday, January 20, 2015, Ryan Dunckel <notifications@github.com> wrote:

> In scripts/repl.js
> <https://github.com/6to5/6to5.github.io/pull/61#discussion-diff-23255084>:
>
> > @@ -31,8 +31,10 @@
> >    var getQuery = function() {
> >      var query = window.location.hash.replace(/^\#\?/, '');
> >      return _.transform(query ? query.split('&') : null, function(result, val) {
> > -      val = val.split('=');
> > -      result[val[0]] = decodeURIComponent('' + val[1]);
> > +      var equalsIndex = val.indexOf('=');
> > +      var name = val.substring(0, equalsIndex);
> > +      var value = val.substring(equalsIndex + 2);
>
> @thejameskyle <https://github.com/thejameskyle> Using the fix from
> @fabiomcosta <https://github.com/fabiomcosta> , I created a fix branch on
> the refactor from #73 <https://github.com/6to5/6to5.github.io/pull/73> at
> d481f0a
> <https://github.com/6to5/6to5.github.io/commit/d481f0ac020ffbf8f82f3c30c01d200841a201f1>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/6to5/6to5.github.io/pull/61/files#r23255084>.
>


-- 
--
Fabio Miranda Costa
*twitter:* @fabiomiranda
*github:* fabiomcosta